src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -561,8 +676,8 @@ private void writeMatrix() {
             writer.write(CROSSCHECK_BY.name());
 
             // write the names of the keys as the first row
-            for (int col = 0; col < rhsMatrixKeys.size(); col++) {
-                writer.write('\t' + rhsMatrixKeys.get(col));
+            for (String rhsMatrixKey : rhsMatrixKeys) {
+                writer.write('\t' + rhsMatrixKey);
```

Comment:
Can you make this same change below for `lhsMatrixKeys`?

Reply:
I can only do it for the inner loop.